jvm/src/main/kotlin/org/bibletranslationtools/maui/jvm/mappers/FileVerifier.kt

```diff
@@ -0,0 +1,90 @@
+package org.bibletranslationtools.maui.jvm.mappers
+
+import javafx.beans.property.SimpleObjectProperty
+import org.bibletranslationtools.maui.common.data.FileStatus
+import org.bibletranslationtools.maui.common.data.Grouping
+import org.bibletranslationtools.maui.common.data.VerifiedResult
+import org.bibletranslationtools.maui.common.io.Versification
+import org.bibletranslationtools.maui.jvm.ui.FileDataItem
+import org.wycliffeassociates.otter.common.audio.wav.CueChunk
+import org.wycliffeassociates.otter.common.audio.wav.WavFile
+import org.wycliffeassociates.otter.common.audio.wav.WavMetadata
+import tornadofx.isInt
+
+class FileVerifier(private val versification: SimpleObjectProperty<Versification>) {
```

Comment:
I'm not sure why we need a javafx beans property here for versification. This introduces the unsafe non-null cast `!!` you have below.